app/models/metric/chargeback_helper.rb

```diff
@@ -21,11 +21,11 @@ def tag_prefix
   def tag_list_reconstruct
     tag_list = tag_names
 
-    if resource_type == Container.name
+    if resource.instance_of?(Container)
```

Comment:
I think this may lead to extra `Select` query.

Although, this is minor nitpick and can be ignored.